lib/resources/r4/document_reference.yaml

```diff
@@ -29,3 +100,193 @@ fields:
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author
+  example: |
+    {
+      "author": [
+        {
+          "reference": "Practitioner/2150097"
+        }
+      ]
+    }
+
+- name: authenticator
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Which person authenticates that this document is valid.
+  note: If set, must be a Practitioner reference with valid millennium id. Only supported with system access. Cannot be populated for provider access.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator
+  example: |
+    {
+      "authenticator": {
+        "reference": "Practitioner/21500981"
+      }
+    }
+
+- name: content
+  required: 'Yes'
+  type: BackboneElement
+  action:
+    - create
+  description: The document and format referenced.
+  note: Must contain only one entry
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content
+  example: |
+   {
+      "content": [
+        {
+          "attachment": {
+            "contentType": "text/xml;charset=utf-8",
+            "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
+            "title": "title",
+            "creation": "2020-07-29T21:02:04.000Z"
+          }
+        }
+      ]
+   }
+
+- name: content.attachment
+  required: 'Yes'
+  type: Attachment
+  action:
+    - create
+  description: The document or URL of the document along with critical metadata to prove content has integrity.
+  note: Data and content type must be provided.
+  url: https://www.hl7.org/fhir/documentreference-definitions.html#DocumentReference.content.attachment
+  example: |
+   {
+     "attachment": {
+            "contentType": "application/xml;charset=utf-8",
+            "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
+            "title": "title",
+            "creation": "2020-01-15T14:35:29.000Z"
+          }
+   }
+
+- name: content.attachment.contentType
+  required: 'Yes'
+  type: Code
+  action:
+    - create
+  description: Identifies the type of the data in the attachment and allows a method to be chosen to interpret or render the data. Includes mime type parameters such as charset where appropriate.
+  note: Must be provided. Content Type must include a supported MIME type and character set. The supported MIME Types are `application/pdf`, `text/rtf`, `text/html`, `application/xml`, and `application/xhtml+xml`
+  url: https://www.hl7.org/fhir/datatypes-definitions.html#Attachment.contentType
+  example: |
+   {
+     "contentType": "application/xml;charset=utf-8"
+   }
+
+- name: content.attachment.data
+  required: 'Yes'
+  type: base64Binary
+  action:
+    - create
+  description: The actual data of the attachment - a sequence of bytes, base64 encoded.
+  note: must be provided.
```

Comment:
We don't need this note, we already state that it is required.

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/134ae0b5f90e1a42b6d4c1e53e7cd966432c4fee